controllers/jetstream/stream.go

```diff
@@ -148,56 +146,50 @@ func (c *Controller) processStreamObject(str *apis.Stream, jsmc jsmClient) (err
 		servers := spec.Servers
 		if c.opts.CRDConnect {
 			// Create a new client
-			opts := make([]nats.Option, 0)
-			opts = append(opts, nats.Name(fmt.Sprintf("%s-str-%s-%d", c.opts.NATSClientName, spec.Name, str.Generation)))
+			natsCtx := &natsContext{}
+			natsCtx.Name = fmt.Sprintf("%s-str-%s-%d", c.opts.NATSClientName, spec.Name, str.Generation)
```

Comment:
This is going to kill pooling - name is part of the pooling hash

Reply:
Good catch! Fixed here: https://github.com/nats-io/nack/pull/152/commits/0768570c8a0935b8cf0f76d1c776fac30c48e466